UrlRegex.txt

```diff
@@ -1 +1 @@
-^https?:\/\/(?!www\.google\.)(((?!desktop).*.)?(githubusercontent|github)\.(?!io|myshopify)).*
\ No newline at end of file
+^https?:\/\/(?!www\.google\.com)(((?!desktop).*.)?(githubusercontent|github)\.(?!io|myshopify|blog)).*
```

Comment:
Should be `desktop).*\.` instead of `desktop).*.` :)

Reply:
i think you're right :-)